### PR TITLE
[Android/FireTV] Add title overlay

### PR DESCRIPTION
**Pull request by fredrifoUni**
## Test plan

### CODE REVIEW

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -281,6 +282,22 @@ public void run() {
         });
     }
 
+    public void refreshTitle_fork(){
+        if( playerControlView == null ){ return; }
```
Comment (yvbeek) [inline android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:286]: Nit: would be good to keep an eye on formatting. I've seen some irregular spacing.
```java
if (playerControlView == null) { return; }
```

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 2]
```diff
@@ -281,6 +282,22 @@ public void run() {
         });
     }
 
+    public void refreshTitle_fork(){
```
Comment (yvbeek) [inline android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:285]: Not sure if we want to include `_fork` in these names. That might create a lot more work if we plan to PR this to the `react-native-video` team. I think the comments are a good way to mark it though.

**Review by yvbeek — APPROVED** [review]
Looks good!

Don't we also need changes to the TypeScript-side of the implementation? I can imagine that it will start yelling in our implementation that `title` is an unknown prop?

**Comment by fredrifoUni** [comment]
> Looks good!
> 
> Don't we also need changes to the TypeScript-side of the implementation? I can imagine that it will start yelling in our implementation that `title` is an unknown prop?

'title' is already there on typescript. It was just not implemented properly for Android yet

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 1 above)
Comment (fredrifoUni) [reply android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:286]: Yeah, formatting is kinda all over through this project, so it makes it hard to decide on an approach. Changing all the original fork code would be a pain when dealing with merge conflicts. 

I'll start moving towards that formatting in my own changes though

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 2 above)
Comment (fredrifoUni) [reply android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:285]: Added this to the ReadMe under tech debt

**Pull request merged**